src/OpenSage.Game/Logic/Object/Body/ActiveBody.cs

```diff
@@ -39,10 +39,7 @@ public override void SetInitialHealth(float multiplier)
 
         public override void DoDamage(DamageType damageType, Fix64 amount, DeathType deathType, TimeInterval time)
         {
-            if (Health <= Fix64.Zero)
-            {
-                return;
-            }
+            if (Health <= Fix64.Zero) return;
```

Comment:
Please revert this - our convention is to always use braces, including for single line blocks.

Reply:
done